src/screens/OrderDetails/index.tsx

```diff
@@ -91,11 +91,10 @@ class OrderDetails extends React.Component<Props, State> {
               <Text style={styles.label}>Picklist</Text>
             </View>
           </View>
-          <View style={styles.row}>
+          <View style={styles.bottomList}>
             <FlatList
               data={this.state.pickList?.picklistItems}
               ListEmptyComponent={this.renderEmptyContainer()}
-              ListFooterComponent={<View style={styles.bottomList}></View>}
```

Comment:
Why did you change this component? The ticket was meant for InboundOrderList. Was the same issue happening in this component?